src/main/java/frc/robot/subsystems/RollerSubsystem.java

```diff
@@ -231,6 +233,20 @@ private HeldPiece getColorSensorPiece() {
         gEntry.setValue(green);
         bEntry.setValue(blue);
 
+        //if the color is 0 0 0, the sensor is broken and should be rebooted
+        //only do this every couple seconds
+        if(red == 0 && green == 0 && blue == 0){
+            if(!colorTimer.hasStarted()){
+                crolorSensor = new ColorSensorV3(I2C.Port.kMXP);
+                colorTimer.start();
+            } else if (colorTimer.advanceIfElapsed(2)){
+                crolorSensor = new ColorSensorV3(I2C.Port.kMXP);
```

Comment:
don't we also need to reset the timer here, otherwise it'll just spam after the 2nd time